### ISSUE TITLE
Distro signatures for FreeBSD

### ISSUE THREAD

**Issue by nthrow** [cobbler/cobbler#1583]
On importing a few FreeBSD ISOs into 2.6.9 I found the default signatures were failing.  Some hacking and help from the IRC channel, and I've confirmed that the following works for 9.3-10.2:
```
   "9.3": {
    "signatures":["boot"],
    "version_file":"device\\.hints",
    "version_file_regex":"^.*FreeBSD: releng/9.3(.*)/sys/(.*)/conf/GENERIC.hints.*$",
    "kernel_arch":"device\\.hints",
    "kernel_arch_regex":"^.*FreeBSD: release/9.3(.*)/sys/(.*)/conf/GENERIC.hints.*$",
    "supported_arches":["i386","amd64"],
    "supported_repo_breeds":[],
    "kernel_file":"pxeboot(.*)",
    "initrd_file":"mbr",
    "isolinux_ok":false,
    "default_kickstart":"",
    "kernel_options":"",
    "kernel_options_post":"",
    "boot_files":[]
   },
   "10.0": {
    "signatures":["boot"],
    "version_file":"freebsd-version",
    "version_file_regex":"USERLAND_VERSION=\"10.0-RELEASE\"",
    "kernel_arch":"device\\.hints",
    "kernel_arch_regex":null,
    "supported_arches":["i386","amd64"],
    "supported_repo_breeds":[],
    "kernel_file":"base.txz",
    "initrd_file":"base.txz",
    "isolinux_ok":false,
    "default_kickstart":"",
    "kernel_options":"",
    "kernel_options_post":"",
    "boot_files":[]
   },
   "10.1": {
    "signatures":["boot"],
    "version_file":"freebsd-version",
    "version_file_regex":"USERLAND_VERSION=\"10.1-RELEASE\"",
    "kernel_arch":"device\\.hints",
    "kernel_arch_regex":null,
    "supported_arches":["i386","amd64"],
    "supported_repo_breeds":[],
    "kernel_file":"base.txz",
    "initrd_file":"base.txz",
    "isolinux_ok":false,
    "default_kickstart":"",
    "kernel_options":"",
    "kernel_options_post":"",
    "boot_files":[]
   },
   "10.2": {
    "signatures":["boot"],
    "version_file":"freebsd-version",
    "version_file_regex":"USERLAND_VERSION=\"10.2-RELEASE\"",
    "kernel_arch":"device\\.hints",
    "kernel_arch_regex":null,
    "supported_arches":["i386","amd64"],
    "supported_repo_breeds":[],
    "kernel_file":"base.txz",
    "initrd_file":"base.txz",
    "isolinux_ok":false,
    "default_kickstart":"",
    "kernel_options":"",
    "kernel_options_post":"",
    "boot_files":[]
   }
  },
```

**Comment by jmaas** [comment]
These are now incorporated into the code for the 2.6, 2.8 (not released) and 3.0 (not released) branches. Online signature update feature has these signatures for 2.4, 2.6, 2.8 and 3.0

**Comment by jmaas** [comment]
Thanks for contributing !!!!
Very much appreciated :+1:
